src/utils/formatNumbers.ts

```diff
@@ -33,3 +33,44 @@ export const currencyFormatter = (
     return formatter.format(value);
   };
 };
+
+export const dateFormatter = (
```

Comment:
Seems to be very similar to formatDate.ts and one is using translation, the other is not, would be good to be consistent (name of the file might lead to some questions as well)

Reply:
good feedbacks. thanks